source/Notation/General.lagda

```diff
@@ -236,14 +237,14 @@ Notice that reasoning in compositional order with g : B → C and f : A → B
  B ∘⟨ f ⟩
  A ▢
 
-amounts to normal composition, but importantly a function A → C in the 'bottom
-up' direction. This may seems strange at first, as one might expect this feature
-to only be useful in the forward direction, that is in diagrammatic order. In
-fact, the above actually reflects a common mode of proof where one proves C by
-observing it suffices to prove B (and supplying a map from B to C) and then
-proving B by observing it suffices to prove A (and supplying a map from A to B).
-For this reason we provide notation that allows us to display proofs of this
-sort.
+amounts to a function A → C (via normal composition), but it appears in the
+'bottom up' direction. This may seem strange at first, as one might expect
+this feature to only be useful in the forward direction, that is in
```

Comment:
Minor grammatical point: there should be a comma after *that is* to indicate that it's used as in *i.e*.